### PR TITLE
Minor fixes: fixed cancancan errors , corrected grammar [#170]

### PR DESCRIPTION
**Pull request by meenakshikumari**

### CODE REVIEW

File: app/views/order_mailer/send_place_order_reminder.html.haml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 %p Hello #{@name},
 %p 
   This is just for the reminder that you can place order only upto 
-  %b #{@end_time}
+  %b 12:30 PM
```
Comment (gautamrege) [inline app/views/order_mailer/send_place_order_reminder.html.haml]: Hard coding? why

File: app/controllers/menu_items_controller.rb [hunk 2]
```diff
@@ -75,11 +67,11 @@ def load_company
   end
 
   def load_terminal
-    @terminal = @current_company.terminals.find params[:terminal_id]
+    @terminal = Terminal.find params[:terminal_id]
```
Comment (gautamrege) [inline app/controllers/menu_items_controller.rb]: Why was this changed? 

Terminals should be scoped to `@current_company`

File: app/views/order_mailer/send_place_order_reminder.html.haml (hunk 1 above)
Comment (meenakshikumari) [inline app/views/order_mailer/send_place_order_reminder.html.haml]: This was told by Seema ma'am. She told to keep this value static for now.

File: app/controllers/menu_items_controller.rb (hunk 2 above)
Comment (meenakshikumari) [inline app/controllers/menu_items_controller.rb]: @current_company was instantiated in load_company method. This method was skipped for some of the functions in terminals' controller. So, this scope was removed.

**Comment by meenakshikumari** [comment]
This pull request is duplicated in another pull request [#186] because of dependency. So i'm closing this pull request

**Comment by meenakshikumari** [comment]
This pull request is duplicated in another pull request [#186] because of dependency. So i'm closing this pull request
